src/Microsoft.VisualBasic.Core/src/Microsoft/VisualBasic/FileSystem.vb

```diff
@@ -1369,6 +1378,9 @@ Namespace Microsoft.VisualBasic
                         Throw VbMakeException(vbErrors.IllegalFuncCall)
                 End Select
             End If
+#Else
+            Throw New PlatformNotSupportedException()
```

Comment:
You'll ideally want to have Unix tests that verify it throws PNSE (as oopposed to crashing somehow)